miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -73,16 +73,25 @@ internal class MiniAppWebView(
         loadUrl(getLoadUrl())
     }
 
-    @Suppress("TooGenericExceptionCaught", "SwallowedException")
-    fun destroyView() = try {
+    override fun onAttachedToWindow() {
+        super.onAttachedToWindow()
+        onResume()
+    }
+
+    override fun onDetachedFromWindow() {
+        super.onDetachedFromWindow()
+
+        onPause()
+        if (!(context as Activity).isDestroyed) {
```

Comment:
Is there the possibility that this cast could fail? Maybe this should be `if (context is Activity && !(context as Activity).isDestroyed)`?

Reply:
I think, we can skip it as we are already have a check called `isContextValid` when providing this web view from `RealMiniAppDisplay`.